composer.json

```
@@ -17,8 +17,7 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
-        "ext-vips": ">=0.1.2",
+        "php": ">=7.4",
```

The FFI extension needs probably to be listed here as well.
```suggestion
        "php": ">=7.4",
        "ext-ffi": "*",
```

---

Oh, good idea, done.